Library/Homebrew/github_packages.rb

```diff
@@ -147,8 +146,6 @@ def upload_bottle(user, token, skopeo, bottle_hash, dry_run:)
     source = "https://github.com/#{org}/#{repo}/blob/#{git_revision}/#{git_path}"
     documentation = if formula.tap.core_tap?
       "https://formulae.brew.sh/formula/#{formula_name}"
-    else
-      formula.tap.remote
```

Comment:
Good catch that `tap.remote` may not be HTTPS. I'd like to link to the third-party tap web page here.

Reply:
There's no such thing.

Reply:
Such as https://github.com/brewsci/homebrew-bio when `tap.remote` is HTTPS.